tools/percolator/rollout_nested_collection.xml

```diff
@@ -0,0 +1,52 @@
+<tool id="rollout_nested_collection" name="Rollout nested list" version="3.1">
+    <description>into a single long list</description>
```

Comment:
Opps - sorry if I indicated this required "the missing operations" - I was probably mistaken.